lib/tasks/amr_importer/asl_copy.rake

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+namespace :amr do
+  desc 'Import data from csv'
+  task asl_copy: :environment do
+    uri = URI(ENV.fetch('ASL_SFTP_URI'))
+    bucket = ENV.fetch('AWS_S3_AMR_DATA_FEEDS_BUCKET')
+    s3 = Aws::S3::Client.new
+    key = s3.get_object(bucket:, key: 'asl_holdings_ftp_key.pem').body.read
+    config = AmrDataFeedConfig.find_by(identifier: 'asl-centrica-solar')
+    Net::SFTP.start(uri.host, uri.user, port: uri.port, key_data: [key], passphrase: ENV.fetch('ASL_SFTP_PASSPHRASE'),
+                                        logger: Rails.logger, non_interactive: true) do |sftp|
+      sftp.dir.glob(uri.path, '*.csv').sort_by { |e| e.attributes.mtime }.reverse_each do |entry|
```

Comment:
The SFTP server has a single folder on login: `Elster`. That contains `Elster/Report` which has the data as CSV but also `Elster/ReportErrors` which contains error logs that aren't relevant to us. These are also in CSV format. Need to avoid fetching those as they're just trigger loading errors.

Reply:
Or am I missing something in the `uri` path?

Reply:
yes uri path is Elster/Report